doc/classes/PopupMenu.xml

```diff
@@ -186,7 +186,7 @@
 			<param index="1" name="submenu" type="String" />
 			<param index="2" name="id" type="int" default="-1" />
 			<description>
-				Adds an item that will act as a submenu of the parent [PopupMenu] node when clicked. The [param submenu] argument is the name of the child [PopupMenu] node that will be shown when the item is clicked.
+				Adds an item that will act as a submenu of the parent [PopupMenu] node when clicked. The [param submenu] argument must be the name of an existing [PopupMenu] that has been previously created and added as a child to the parent [PopupMenu] node. This submenu will be shown when the item is clicked or hovered for long enough.
```

Comment:
```suggestion
				Adds an item that will act as a submenu of the parent [PopupMenu] node when clicked. The [param submenu] argument must be the name of an existing [PopupMenu] that has been added as a child to this node. This submenu will be shown when the item is clicked or hovered for long enough.
```

Existing already implies it's been previously created, and we can clarify that it needs to be a child of this node, and not just any other popup menu.

Also @Calinou edited his suggestion a bit, it now reads 

> This submenu will be shown when the item is clicked, hovered for long enough, or activated using the [code]ui_select[/code] or [code]ui_right[/code] input actions.

I think it's a good idea to include it fully.

Reply:
Thanks！